Source/Plugins/OrientationAnalysis/OrientationAnalysisFilters/ReadCtfData.cpp

```diff
@@ -533,12 +542,19 @@ void ReadCtfData::copyRawEbsdData(CtfReader* reader, QVector<size_t>& tDims, QVe
 
     for(size_t i = 0; i < totalPoints; i++)
     {
-      cellEulerAngles[3 * i] = f1[i];
-      cellEulerAngles[3 * i + 1] = f2[i];
-      cellEulerAngles[3 * i + 2] = f3[i];
-      if(m_CrystalStructures[cellPhases[i]] == Ebsd::CrystalStructure::Hexagonal_High)
+      cellEulerAngles[3 * i] = f1[i] ;
+      cellEulerAngles[3 * i + 1] = f2[i] ;
+      cellEulerAngles[3 * i + 2] = f3[i] ;
+      if(m_CrystalStructures[cellPhases[i]] == Ebsd::CrystalStructure::Hexagonal_High && m_EdaxHexagonalAlignment)
+      {
+        cellEulerAngles[3 * i + 2] = cellEulerAngles[3 * i + 2] + (30.0); // See the documentation for this correction factor
+      }
+      // Now convert to radians if requested by the user
+      if(m_DegreesToRadians)
       {
-        cellEulerAngles[3 * i + 2] = cellEulerAngles[3 * i + 2] + (30.0);
+        cellEulerAngles[3 * i] = cellEulerAngles[3 * i] * SIMPLib::Constants::k_PiOver180;
+        cellEulerAngles[3 * i + 1] = cellEulerAngles[3 * i + 1] * SIMPLib::Constants::k_PiOver180;
+        cellEulerAngles[3 * i + 2] = cellEulerAngles[3 * i + 2] * SIMPLib::Constants::k_PiOver180;;
```

Comment:
There are two semicolons at the end of this line.

Reply:
I am going to just clang-format the entire header and source file.